src/main/java/duke/common/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke.common;
```

Comment:
Don't you think it is a good idea to have more exceptions under DukeExceptions to help with different types of exceptions? I think there can be more classes inheriting from this class!